### ISSUE TITLE
with-react-i18next fails with react-i18next 8.0.4 (latest)

### ISSUE THREAD

**Issue by Nelrohd** [zeit/next.js#5352]
# Examples bug report

## Example name
Example name is `with-react-i18next`.

## Describe the bug
When using `react-i18next` `v8.0.4` in the demo, we get a 500 server error:
```
TypeError: Cannot set property 'wait' of undefined
```
Note that `v8.0.4` is the latest release of react-i18next at this moment.

## To Reproduce
Steps to reproduce the behavior, please provide code snippets or a repository:
1. Clone and install the example
2. Install `react-i18next` with the version `8.0.4`
3. Run the example
4. Open your browser and see the error

## Expected behavior
The page should be displayed.

## Screenshots
None.

## System information
 - OS: macOS High Sierra Version 10.13.6
 - Browser (if applies) Google Chrome Version 69.0.3497.100 (Official Build) (64-bit)
 - Version of Next.js: latest
```
{
  "name": "react-i18next-nextjs-example",
  "version": "1.0.0",
  "description": "",
  "main": "server.js",
  "scripts": {
    "dev": "node server.js",
    "build": "next build",
    "start": "NODE_ENV=production node server.js"
  },
  "author": "",
  "license": "MIT",
  "dependencies": {
    "express": "4.16.3",
    "i18next": "11.9.0",
    "i18next-browser-languagedetector": "2.2.3",
    "i18next-express-middleware": "1.4.0",
    "i18next-node-fs-backend": "2.1.0",
    "i18next-xhr-backend": "1.5.1",
    "next": "latest",
    "react": "^16.5.2",
    "react-dom": "^16.5.2",
    "react-i18next": "8.0.4"
  }
}
```

## Additional context
None.

**Comment by ryuken** [comment]
its broken from version 8.0.0

**Comment by jamuhl** [comment]
could you please try the sample here: https://github.com/i18next/react-i18next/tree/master/example/nextjs

just updated that to v8.0.6 of react-i18next

**Comment by Enalmada** [comment]
@Nelrohd I assume this issue can be closed now? (and #5368 should be merged in to prevent others from getting same error!)

**Comment by Nelrohd** [comment]
Yes, we can close it with merge @Enalmada ! Thank you!
